WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -372,6 +681,194 @@ - (void)configureTodaysStatsCell:(InsightsTodaysStatsTableViewCell *)cell
+    
+    UILabel *label = (UILabel *)[cell.contentView viewWithTag:100];
+    label.text = headerText;
+    
+    cell.bottomBorderEnabled = NO;
+}
+
+
+- (void)configureSectionTwoColumnHeaderCell:(StatsStandardBorderedTableViewCell *)cell withStatsSection:(StatsSection)statsSection
+{
+    StatsGroup *statsGroup = [self statsDataForStatsSection:statsSection];
+    StatsItem *statsItem = [statsGroup statsItemForTableViewRow:2];
+    
+    NSString *leftText = statsGroup.titlePrimary;
+    NSString *rightText = statsGroup.titleSecondary;
+    
+    // Hide the bottom border if the first row is expanded
+    cell.bottomBorderEnabled = !statsItem.isExpanded;
+    
+    UILabel *label1 = (UILabel *)[cell.contentView viewWithTag:100];
+    label1.text = leftText;
+    
+    UILabel *label2 = (UILabel *)[cell.contentView viewWithTag:200];
+    label2.text = rightText;
+}
+
+
+- (void)configureSectionGroupTotalCell:(UITableViewCell *)cell withStatsSection:(StatsSection)statsSection andTotal:(NSString *)total
+{
+    NSString *title;
+    StatsSubSection selectedSubsection = [self statsSubSectionForStatsSection:statsSection];
+    
+    switch (selectedSubsection) {
+        case StatsSubSectionFollowersDotCom:
+            title = [NSString stringWithFormat:NSLocalizedString(@"Total WordPress.com Followers: %@", @"Label of Total count of WordPress.com followers with value"), total];
+            break;
+        case StatsSubSectionFollowersEmail:
+            title = [NSString stringWithFormat:NSLocalizedString(@"Total Email Followers: %@", @"Label of Total count of email followers with value"), total];
+            break;
+        default:
+            break;
+    }
+    
+    UILabel *label = (UILabel *)[cell.contentView viewWithTag:100];
+    label.text = title;
+}
+
+
+- (void)configureSectionGroupSelectorCell:(StatsStandardBorderedTableViewCell *)cell withStatsSection:(StatsSection)statsSection
+{
+    NSArray *titles;
+    NSInteger selectedIndex = 0;
+    StatsSubSection selectedSubsection = [self statsSubSectionForStatsSection:statsSection];
+    
+    switch (statsSection) {
+        case StatsSectionComments:
+            titles = @[NSLocalizedString(@"By Authors", @"Authors segmented control for stats"),
+                       NSLocalizedString(@"By Posts & Pages", @"Posts & Pages segmented control for stats")];
+            selectedIndex = selectedSubsection == StatsSubSectionCommentsByAuthor ? 0 : 1;
+            break;
+        case StatsSectionFollowers:
+            titles = @[NSLocalizedString(@"WordPress.com", @"WordPress.com segmented control for stats"),
+                       NSLocalizedString(@"Email", @"Email segmented control for stats")];
+            selectedIndex = selectedSubsection == StatsSubSectionFollowersDotCom ? 0 : 1;
+            break;
+        default:
+            break;
+    }
+    
+    UISegmentedControl *control = (UISegmentedControl *)[cell.contentView viewWithTag:100];
+    cell.bottomBorderEnabled = NO;
```

Comment:
I want to refactor out these cell configs to the actual class. So for now I'll leave it with the intention of getting rid of the viewWithTag crap.